schlib/comparelibs.py

```diff
@@ -144,23 +146,57 @@ def KLCCheck(lib, component):
 
             if args.verbose:
                 printer.yellow("Changed symbol '{lib}:{name}'".format(lib=lib_name, name=cmp))
+            if args.design_breaking_changes:
+                pins_moved = 0
+                nc_pins_moved = 0
+                pins_missing = 0
+                nc_pins_missing = 0
+                for pin_old in old_cmp[cmp].pins:
+                    pin_new = new_cmp[cmp].getPinByNumber(pin_old['num'])
+                    if pin_new is None:
+                        if pin_old['electrical_type'] == 'N' and pin_new['electrical_type'] == 'N':
+                            nc_pins_missing +=1
+                        else:
+                            pins_missing += 1
+                        continue
+
+                    if pin_old['posx'] != pin_new['posx']:
+                        if pin_old['electrical_type'] == 'N' and pin_new['electrical_type'] == 'N':
+                            nc_pins_moved +=1
+                        else:
+                            pins_moved += 1
+                    elif pin_old['posy'] != pin_new['posy']:
```

Comment:
Since you're not checking if pins moved X or Y, you could combine the "posx" and "posy" clauses. Or does it matter?

Reply:
Done